### PR TITLE
Reason phrases

### PR DESCRIPTION
**Pull request by mikel00per**
I think it would be good for everyone to add a new interface that has the "reason phrases" related to the statusCodes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ Install by adding the package as a [Composer](https://getcomposer.org)
 requirement:
 
 ```bash
-$ composer require fig/http-message-util
+$ composer require mikel00per/http-message-util
```
Comment (shadowhand) [inline README.md:16]: 👎🏼

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 This repository holds utility classes and constants to facilitate common
 operations of [PSR-7](https://www.php-fig.org/psr/psr-7/); the primary purpose is
 to provide constants for referring to request methods, response status codes and
-messages, and potentially common headers.
+messages, reason phrases and potentially common headers.
```
Comment (shadowhand) [inline README.md:6]:
```suggestion
messages, reason phrases, and potentially common headers.
```

**Review by shadowhand — CHANGES_REQUESTED** [review]

File: composer.json [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
     "keywords": ["psr", "psr-7", "http", "http-message", "request", "response"],
+    "homepage": "https://github.com/mikel00per/http-message-util",
     "license": "MIT",
-    "authors": [
+        "authors": [
         {
-            "name": "PHP-FIG",
-            "homepage": "https://www.php-fig.org/"
+            "name": "Miguel Morillo",
+            "email": "miguelmwilliams@gmail.com"
```
Comment (Jean85) [inline composer.json:10]: This is not acceptable. Please rework your PR.

**Review by Jean85 — CHANGES_REQUESTED** [review]
This PR as-is is not acceptable. You would need to talk to the PSR editor and come to our mailing list to discuss such a modification. You're welcome to: https://groups.google.com/g/php-fig

**Comment by mikel00per** [comment]
Sorry, closed. My bad.

**Pull request closed without merging**